react/Viewer/ImageLoader.jsx

```diff
@@ -99,17 +99,24 @@ export class ImageLoader extends React.Component {
         })
       }
     } catch (e) {
-      logger.info(e)
+      logger.error(e)
       this.loadNextSrc(e)
     }
   }
 
   async loadFallback() {
     this.setState({ status: LOADING_FALLBACK })
-    const { file } = this.props
+    const { file, client } = this.props
 
     try {
-      const src = await this.getDownloadLink(this.getFileId(file))
+      if (file.class === 'pdf') {
+        throw new Error('No pdf files fallback')
+      }
```

Comment:
should be tested to describe the behavior. It's hard to understand why this change at first

Reply:
`ImageLoader` n'a pas de test (encore...) à proprement parlé, il est "testé" via `PdfMobileViewer` qui l'appel à un moment donné. Peut-être simplement rajouter pour l'instant un commentaire pour expliquer un peu le pourquoi du if ?

Reply:
Testing that a specific method of a component throws an error in some case it's not very hard to write, no? 

OK for adding a comment

Reply:
C'est un peu plus compliqué que ça car on passe dans une première boucle `loadLink` qui, en cas d'erreur et quelque soit l'erreur fallback sur `loadFallback` qui, si elle rencontre une erreur à son tour, retourne finalement l'erreur et set le status du composant en `failed`.